common/app/experiments/ExperimentsDefinition.scala

```diff
@@ -31,6 +37,16 @@ trait ExperimentsDefinition {
 
   def isControl(experiment: Experiment)(implicit request: RequestHeader): Boolean =
     isIn(experiment)(_.isControl)
+
+  def groupFor(experiment: Experiment)(implicit request: RequestHeader): Participation = {
+    if(isParticipating(experiment)){
+      Participant
+    } else if(isControl(experiment)){
```

Comment:
There are a couple of minor whitespace issues here (if needs a space between itself and the open parens). A reminder to myself that we should probably add Scalariform to this project sooner rather than later!

Reply:
do you mean ```if( x )``` or ```if (x)``` or ```if ( x )```, the whitespace in the existing ExperimentsDefinition uses ```if(x)```